src/PagesBody/UserDogsPage/UserDogPage.js

```diff
@@ -0,0 +1,33 @@
+import ClassOrDogButton from '../../Molecules/ClassOrDogButton/ClassOrDogButton';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import MainButton from '../../Atoms/MainButton/MainButton';
+import participants from '../../Data/MongoDBMock/participants';
+import { useNavigate } from 'react-router-dom';
+
+const UserDogPage = () => {
+  const navigate = useNavigate();
+
+  return (
+    <ColumnWrapper paddingLeftRight={1} paddingTop={0.5}>
+      {participants[0].dogs.map((dog, index) => {
```

Comment:
you can change this `participants[0].dogs` to reading data from Context:

![image](https://user-images.githubusercontent.com/75560322/152641678-d8e124ca-8e0f-4857-baf4-6b743323b4ee.png)